video/out/w32_common.c

```diff
@@ -128,6 +128,9 @@ struct vo_w32_state {
     // UTF-16 decoding state for WM_CHAR and VK_PACKET
     int high_surrogate;
 
+    // Whether to fit the window on screen on next window state updating
+    bool fit_on_screen;
```

Comment:
Fine for now, but in future it might be better to solve this by applying the fit-on-screen logic to prev_windowrc when it's updated.

Reply:
Maybe, but it'll require storing the virtual screen size in a separate RECT, i.g. `vscreenrc`, instead of `screenrc`, for the case when `--fs-screen=all` is set. I still think that the fit on screen logic should stay as it is now, to make sure the window styles are up to date at the moment of adjusting the window bounds.